spec/example_spec.rb

```diff
@@ -0,0 +1,5 @@
+describe MigrationLogController do
+  it "stuff" do
+    expect(1).to eq 1
+  end
+end
```

Comment:
Is this file required?

Reply:
It was my placeholder for your tests otherwise travis would have nothing to run.

Reply:
Ah, ok, thanks.